samples/scala-value-entity-customer-registry/src/test/scala/customer/domain/CustomerValueEntitySpec.scala

```diff
@@ -0,0 +1,53 @@
+/* This code was generated by Akka Serverless tooling.
```

Comment:
This looks unrelated though?

Reply:
yeah that was me running a test, I'll remove it from the PR.

Reply:
removed.